Assets/2DGamekit/Art/Sprites/Environment/BGPlants_0.controller

```diff
@@ -0,0 +1,72 @@
+%YAML 1.1
```

Comment:
These animations should probably be in a folder separate from the 2D Game Kit assets. I'll probably move them if it's not too difficult.